tests/parser/WellTests.cpp

```diff
@@ -944,27 +916,6 @@ BOOST_AUTO_TEST_CASE(WCH_Rates_NON_Defaulted_VFP)
     BOOST_CHECK_EQUAL(p.BHPLimit, 101325.);
 }
 
-BOOST_AUTO_TEST_CASE(WCH_Whistctl)
-{
```

Comment:
I have not looked into details. There are some tests removed. What is the reason?

Reply:
The tests which have been removed tested non-intuitive interaction between the `WellProductionProperties` and `Schedule` objects - with this PR that interaction has been significantly simplified / removed - and there is really not anything to test.